packages/ephemeral/service.js

```diff
@@ -48,7 +51,27 @@ module.exports = declareInjections({
         let schemaTypes = this.schemaLoader.ownTypes();
         for (let model of initialModels) {
           let isSchema = schemaTypes.includes(model.type);
-          storage.store(model.type, model.id, model, isSchema);
+          if (model.readable) {
+            // store binary data
+            let id = model.id || crypto.randomBytes(20).toString('hex'); // if model has id, use it, otherwise generate one
+            let blob = await streamToPromise(model);
+            let storedDocument = {
+              type: 'cardstack-files',
+              id,
+              attributes: {
+                'created-at':   new Date().toISOString(),
```

Comment:
This may be better as `statSync(model.path).ctime`

Reply:
(Assuming stat is OK in general here, I'm going to write another comment about that.